spec.md

```diff
@@ -445,9 +445,99 @@ Resource properties:
 * `name` - optional
 * Links to any metadata objects controlled by the DataPub server
 * Links to any metadata objects modifiable by the resource's controller
-### Resource Operations
 
-#### Resource HTTP API
+### Create Resource Operation
+
+A resource is created by making a Create Resource request to a given [=space=],
+and (optionally) a [=collection=]. If a collection is not explicitly specified,
+the resource is created in that space's [=default collection=].
+
+Using the HTTP API, a resource can be created either by making a POST request
+to a space and collection, or by making a PUT request to a specific resource
+(see the Update Resource operation).
+
+#### (HTTP API) POST `/spaces/{space_id}/{collection/*}`
+
+* Requires appropriate authorization
+  - Invoking this method via [ZCAP](#zcap) requires a capability allowing the
+    [`POST` action](#post-action) on the collection
```

Comment:
there are other requirements, e.g. the capability chain being rooted in a resource controller

Reply:
I think that's implied by "appropriate authorization". I didn't want to get into authz details in the method definitions.

Reply:
That said, I updated that line to match the other sections (so, should fulfill your request)